### ISSUE TITLE
class mapping and training loss

### ISSUE THREAD

**Issue by ChunmianLin** [fizyr/keras-retinanet#1408]
Hi, I have a tricky problem about class mapping of CSV data. Why the id begins from 0?
Generally, the class id is from 1 to 'class_num', and 0 indicates the background, is it?
So, this setting confuses me a lot, and I cannot understand.

Could you please check my problem? Thanks in advance!

**Comment by ChunmianLin** [comment]
Another problem confused me is that the training loss begins at a small value (about 3.7-4.0) at the first epoch, and converges quickly at the following step (after one epoch, loss is only less than 1.5). Is it OK?
BTW, the classification loss is always small, does it  hint the model cannot learn useful information from data?
Please **help me**!!! I think I **stuck in model training and fine-tuning work.**

Thanks in advance!!!
